geowebcache/core/src/main/java/org/geowebcache/util/HttpClientBuilder.java

```diff
@@ -36,17 +33,17 @@
 
     private UsernamePasswordCredentials httpcredentials = null;
 
-    private UsernamePasswordCredentials proxycredentials = null;
-
     private AuthScope authscope = null;
 
-    private URL proxyUrl = null;
-
     private Integer backendTimeoutMillis = null;
+    private static final HttpClientConnectionManager connectionManager =
+            new PoolingHttpClientConnectionManager();
 
     private boolean doAuthentication = false;
 
-    private int concurrency;
```

Comment:
Hum... was this configurable before? Yep, it's a constructor argument, still there but I don't see it being used. Think it can be set in a "connection manager" class (or something like that, vague memories)

Reply:
good catch, I knew there was something I needed to come back to